beacon-chain/rpc/beacon/blocks.go

```diff
@@ -192,6 +194,18 @@ func (bs *Server) StreamBlocks(_ *ptypes.Empty, stream ethpb.BeaconChain_StreamB
 					// One nil block shouldn't stop the stream.
 					continue
 				}
+				if featureconfig.Get().NewStateMgmt {
+					headState, err := bs.HeadFetcher.HeadState(bs.Ctx)
```

Comment:
Dont need `NewStateMgmt` for `bs.HeadFetcher.HeadState(`

Reply:
thanks